python/src/main/db.py

```diff
@@ -30,7 +30,7 @@ def insert_person(self, person):
             logging.error("Key '%s' was not unique and hasn't been entered into the db" % person.email)
 
     def insert_item(self, item):
-        self.cursor.execute("INSERT INTO Item VALUES('%s', '%s', '%d', '%d', '%s')"
+        self.cursor.execute("INSERT INTO Item VALUES('%s', '%s', '%s', '%d', '%s')"
```

Comment:
why have you changed to string here?

Reply:
Was initially to try fix a bug, but then thought actually makes sense to be able to answer "40ish" or something instead of strictly a number

Reply:
Why does that make sense?

Reply:
Okay so I'm adding to my list. I know I want some new Nike clothes, but don't know if I want trousers, shirt, or whatever. Just putting it as a suggestion. Having a string as opposed to a number means I can give an estimate or vague idea of a price, so the buyers can rule themselves in/out buying it accordingly